src/App.jsx

```diff
@@ -0,0 +1,13 @@
+import './App.css'
+import Message from './components/greeting'
+
+function App() {
+
+  return (
+    <>
+      <Message />
+    </>
+  )
+}
```

Comment:
- [ ] Kindly setup your routes with react-router, currently your app works well but the requirement requires you  to set up a route  that leads to your greeting component, please update your app 👍🏿